core/tests/data/dummy_ci_tests/dummy_protractor.conf.js

```diff
@@ -0,0 +1,9 @@
+var suites = {
```

Comment:
Put a boilerplate line like `var path = require('path')` at the top to ensure that the script parsing is robust to that.

Ditto in the next file.

Reply:
Done. But can you please explain why this is required? I don't get it.